basyx/aas/adapter/json/json_deserialization.py

```diff
@@ -70,7 +70,7 @@ def _get_ts(dct: Dict[str, object], key: str, type_: Type[T]) -> T:
     return val
 
 
-def _expect_type(object_: object, type_: Type, context: str, failsafe: bool) -> bool:
+def _is_of_type(object_: object, type_: Type, context: str, failsafe: bool) -> bool:
```

Comment:
I think `_expect_type()` fits the purpose of this function better, as it doesn't just check the type, but also raises an exception in non-failsafe mode, so it expects the type.

Reply:
`_expect_type()` has some imperative mood, as if the object should receive the type as a typehint. It may probably be misunderstood.

May be something like `_validate_type()` or `_validate_object_type()` would be ok.